src/types.rs

```diff
@@ -401,13 +403,15 @@ impl Rewrite for ast::WherePredicate {
                     };
                     let used_width = type_str.len() + colon.len();
                     let budget = try_opt!(shape.width.checked_sub(used_width));
-                    let bounds_str: String = try_opt!(bounds
-                        .iter()
-                        .map(|ty_bound| {
-                            ty_bound
-                                .rewrite(context, Shape::legacy(budget, shape.indent + used_width))
-                        })
-                        .collect::<Option<Vec<_>>>())
+                    let bounds_str: String = try_opt!(
+                        bounds
+                            .iter()
+                            .map(|ty_bound| {
+                                ty_bound.rewrite(context,
+                                                 Shape::legacy(budget, shape.indent + used_width))
+                            })
+                            .collect::<Option<Vec<_>>>()
+                    )
```

Comment:
this seems unfortunate, especially as the very similar example below looks better